src/Microsoft.AspNetCore.TestHost/TestServer.cs

```diff
@@ -49,16 +70,26 @@ public IWebHost Host
         {
             get
             {
+                if (_hostInstance == null)
```

Comment:
nit: stay consistent with getter syntax. This one checks _hostInstance == null while other uses ??.